### DISCUSSION TITLE
How do I handle secrets in a public repo?

### DISCUSSION THREAD

**Discussion by jrgdnn12** [jrgdnn12/RemitEase#20] [General]
Don't say change the repo visibility, that cost us money.

I have fired up the MySQL dev and prod instances. I'm puzzled as to how I can share the secrets with other stakeholders.

Against all common sense, and considering that this is a class project. I will publish the credential for the database in a txt file. 

To reduce insecurity, I will be whitelisting UNLV's IPV4. And so this should be the only place you can talk to the database. 

Until we find a better solution.

JA

**Comment by phildeemed56** [comment]
Sounds good.

I am almost done with the creation of the FXML files.

**Reply by jrgdnn12** [reply]
When you get a chance please take a look at PR-19.

**Comment by jrgdnn12** [comment]
So computer systems that connect to the internet thorough NSHE's Nevada net can use more than one public IPV4, I was kind of expecting this.

 Praying that I'm doing the right thing, I have changed the whitelist netmask to /16. 

Now you should be able to connect to UNLV's VPN and talk to the database from anywhere.

To get the VPN first download the FortiClient for your platform here:
https://www.fortinet.com/support/product-downloads#vpn

Then connect with these settings in Remote Access:
VPN: SSL-VPN
Connection Name: UNLV VPN
Remote Gateway : secure.unlv.edu
check on "enable single Sing On (SSO) for VPN tunnel"

save and connect.

You should be greeted by an ACE authentication portal. 


Cheers

JA
